config/bot.yml

```diff
@@ -1,2 +0,0 @@
-SLACK_API_TOKEN: 'change me'
-bot_name: 'bot'
```

Comment:
It was replaced by this one https://github.com/ImperialLabs/slapi/blob/script_container/config/bot.yml